packages/cli/lib/validators/marketplaceValidators/theme/ThemeConfigValidator.js

```diff
@@ -98,6 +100,6 @@ class ThemeValidator extends BaseValidator {
 }
 
 module.exports = new ThemeValidator({
-  name: 'Theme',
-  key: 'theme',
+  name: 'Theme config',
+  key: 'themeConfig',
```

Comment:
Would it make sense to have these in an enum?

Reply:
Good idea, done in 2359c16